pkg/ebpf/c/tracer-ebpf.c

```diff
@@ -853,16 +900,186 @@ int kprobe__tcp_v4_destroy_sock(struct pt_regs* ctx) {
     return 0;
 }
 
+SEC("kprobe/udp_destroy_sock")
+int kprobe__udp_destroy_sock(struct pt_regs* ctx) {
+    struct sock* sk = (struct sock*)PT_REGS_PARM1(ctx);
+
+    if (sk == NULL) {
+        log_debug("ERR(udp_destroy_sock): socket is null \n");
+        return 0;
+    }
+
+    // get tracer status and return early if we are still offset-guessing
+    u64 zero = 0;
+    tracer_status_t* status = bpf_map_lookup_elem(&tracer_status, &zero);
+    if (status == NULL) {
+        return 0;
+    }
```

Comment:
I believe it's safe to delete this whole chunk of code since now we only enable the `kprobes` when offset guessing is done :)